pallets/pools/src/lib.rs

```diff
@@ -447,6 +464,12 @@ pub mod pallet {
 		NoSolutionAvailable,
 		/// One of the runtime-level pool parameter bounds was violated
 		PoolParameterBoundViolated,
+		/// No update for the pool is scheduled
+		NoScheduledUpdate,
+		/// Scheduled time for this update is in the future
+		ScheduledTimeHasNotPassed,
+		/// In the last executed epoch, not all redemptions were fulfilled
+		RedemptionsHaveNotBeenFulfilled,
```

Comment:
Would make this more generic and say `UpdatePrerequesitesNotFulfilled` (In case we are going with my idea from above)